styles/style.css

```diff
@@ -297,3 +307,30 @@ nav > ul > li {
 #projectCards {
   margin-top: 45px;
 }
+
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+form > input {
+  height: 48px;
+}
+
+form > textarea {
+  padding: 10px;
+}
+
+.btnwrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+footer {
```

Comment:
- [ ] There is a page overflow in the X-axis, I don't know if this was there even in other milestones, but it has to be fixed.

![overflow](https://user-images.githubusercontent.com/53124548/134164130-08884b34-78e9-4b93-b8b3-7ff1ca7944b2.png)
![overflow2](https://user-images.githubusercontent.com/53124548/134164147-560e9f83-1c8a-4a00-92ae-988168bcfc5a.png)